plutus-ledger/src/Ledger/Tx/Internal.hs

```diff
@@ -131,6 +131,19 @@ txOutDatumHash (TxOut (C.TxOut _aie _tov tod _rs)) =
     C.TxOutDatumInTx _era scriptData ->
       Just $ datumHash $ Datum $ dataToBuiltinData $ C.toPlutusData scriptData
 
+cardanoTxOutDatum :: TxOut -> Maybe Datum
```

Comment:
You can call this just `txOutDatum`, we use the `cardano` prefix to imply the input or output types are from `cardano-api`.

Reply:
I'm getting the following error that I'm not sure how to resolve when I have the name as `txOutDatum`

`src/Ledger/Tx.hs:20:7: error:
    Conflicting exports for ‘txOutDatum’:
       ‘module Export’ exports ‘Export.txOutDatum’
         imported from ‘Ledger.Tx.Internal’ at src/Ledger/Tx.hs:117:1-35
       ‘module Export’ exports ‘Export.txOutDatum’
         imported from ‘Plutus.V1.Ledger.Tx’ at src/Ledger/Tx.hs:(118,1)-(119,84)
   |
20 |     ( module Export
   |       ^^^^^^^^^^^^^`

Reply:
Does it work it you add `txOutDatum` in the `hiding` list of `Plutus.V1.Ledger.Tx` on line 118/118 of `plutus-ledger/src/Ledger/Tx.hs`?